### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_Remote Run_Keyboard: Unable to switch between two windows at "Remote Spark Job in Cluster" using keyboard

### ISSUE THREAD

**Issue by InduPriya1805** [microsoft/azure-tools-for-java#3590]
**GitHub Tags**: #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #JavatoolingforIntelliJ/Eclipse; #Keyboard; #MAS2.1.1; 

[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.1
Build #IU-192.6262.58

**OS Details:**
OS: Windows (Version 1903, Build: 18362.113)

**MAS Violated:** 2.1.1

**Note:
Similarly after debugging in Debug Remote Spark Job in Cluster section in console tab also we are unable to switch between the windows**

**Repro Steps:**
1)Launch IntelliJ IDEA
2)Tab till create new project-->Azure Spark/HDInsight-->Spark project with Samples(Scala) 
3)Tab till next button and press enter
4)Fill all the details correctly and Tab till finish button and press enter
5)Tab till Project Name-->src-->main-->scala-->sample-->SparkCore_wasbloTest and open it
6)Tab till Edit Configurations menu(select the icon in the upper-right corner)
7)Navigate to Advanced Configuration (Remote Debugging) section and Press the shortcut key to check Enable Spark remote debug
8)Fill Serure Shell (SSH) User Name and SSH password
9)Tab till sparkjobrun and press enter
10)Press Cntrl+Tab to navigate to "Remote Spark Job in Cluster" and verify user is able to switch between the two windows at "Remote Spark Job in Cluster" or not

**Actual Result:**
Unable to switch between two windows at "Remote Spark Job in Cluster" using keyboard

**Expected Result:**
User should be able to  switch between two windows at "Remote Spark Job in Cluster"

**User Impact:**
Keyboard users will be unable to get the information about the logs created if they are unable to navigate using keyboard

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachments:**
[Bug_3590_unable to switch between windows using keyboard.pptx](https://github.com/microsoft/azure-tools-for-java/files/3641364/Bug_3590_unable.to.switch.between.windows.using.keyboard.pptx)

**Comment by t-rufang** [comment]
Note: Can't use tab to swtich between the following two split windows
![image](https://user-images.githubusercontent.com/32627233/65596753-1fb73900-dfca-11e9-9d50-e8c23b056e1b.png)

**Comment by t-rufang** [comment]
This bug is fixed in #3717

**Comment by InduPriya1805** [comment]
Regressed the bug in the below environment
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.2
Build #IC192.6603.28
The bug is fixed so closing the bug. Please find the attached videos for reference.

[Bug_3590_user is able to switch between the tabs2.pptx](https://github.com/microsoft/azure-tools-for-java/files/3819281/Bug_3590_user.is.able.to.switch.between.the.tabs2.pptx)
[Bug_3590_user is able to switch between the tabs1.pptx](https://github.com/microsoft/azure-tools-for-java/files/3819282/Bug_3590_user.is.able.to.switch.between.the.tabs1.pptx)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749693
